netbout-client/src/test/java/com/netbout/client/RtInboxITCase.java

```diff
@@ -65,4 +67,39 @@ public void listsBouts() throws Exception {
         );
     }
 
+    /**
+     * RtInbox can search for bouts.
+     * @throws Exception If there is some problem inside
+     */
+    @SuppressWarnings("unchecked")
+    @Test
+    public void searchesBouts() throws Exception {
+        final Inbox inbox =
+            NbRule.get().aliases().iterate().iterator().next().inbox();
+        final Bout first = inbox.bout(inbox.start());
+        final String firsttitle = "bout1 title";
+        first.rename(firsttitle);
+        first.messages().post("hello");
+        final Bout second = inbox.bout(inbox.start());
+        final String secondtitle = "bout2 title";
+        second.rename(secondtitle);
+        second.messages().post("message with term");
+        final String thirdtitle = "bout title with term";
+        inbox.bout(inbox.start()).rename(thirdtitle);
+        final List<String> titles = new LinkedList<>();
+        for (final Bout bout : inbox.search("term")) {
+            titles.add(bout.title());
+        }
+        MatcherAssert.assertThat(
+            titles,
+            Matchers.hasItems(
+                Matchers.equalTo(secondtitle),
+                Matchers.equalTo(thirdtitle)
+            )
+        );
+        MatcherAssert.assertThat(
+            titles,
+            Matchers.not(Matchers.hasItem(Matchers.equalTo(firsttitle)))
```

Comment:
@CarlosGines I didn't notice that earlier, but you shouldn't need Matchers here, only `firsttitle` should be enough